apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -105,8 +104,10 @@ export function SearchHeader({
         </TouchableOpacity>
       </Animated.View>
       <Animated.View key="icon" style={accountButton}>
-        <ButtonCircle size={50} onPress={goToAccount}>
-          <ContactBubble contact={eAcc} size={50} transparent />
+        <ButtonCircle size={50} onPress={goToQR}>
+          <View style={styles.qrCircle}>
```

Comment:
rename `qrCircle` to eg `circleButton` given that we're now using it for notifications button as well